Vision/Vision.bs

```diff
@@ -93,9 +93,9 @@ Markup Shorthands: markdown yes
 
 # W3C's Vision for the World Wide Web # {#vision-web}
 
-	* The Web is for <a href="https://www.w3.org/TR/ethical-web-principles/#allpeople">all people</a>.
-	* The Web is designed for the <a href="https://www.w3.org/TR/ethical-web-principles/#noharm">good of humanity</a>.
-	* The Web must be <a href="https://www.w3.org/TR/ethical-web-principles/#privacy">secure and respect people's privacy</a>.
+	* The Web is for <a href="https://www.w3.org/TR/ethical-web-principles/#allpeople">all humanity</a>.
+	* The Web is designed for the <a href="https://www.w3.org/TR/ethical-web-principles/#noharm">good of all people</a>.
+	* The Web must be <a href="https://www.w3.org/TR/ethical-web-principles/#privacy">safe to use</a>.
```

Comment:
```suggestion
	* The Web must be <a href="https://www.w3.org/TR/ethical-web-principles/#privacy">safe for all people to use</a>.
```

Reply:
Can you explain what you think this extra phrase adds or clarifies?